.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,128 @@
+name: Bug Report
+description: 'You have noticed a general issue or regression, and would like to report it'
+projects:
+  - jellyfin/jellyfin-web
+labels:
+  - bug
+body:
+  - type: textarea
+    id: bug_description
+    validations:
+      required: true
+    attributes:
+      label: Describe The Bug
+      description: A clear and concise description of what the bug is.
+
+  - type: textarea
+    id: reproduction
+    validations:
+      required: true
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+
+  - type: textarea
+    id: expectations
+    validations:
+      required: false
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+
+  - type: textarea
+    id: logs
+    validations:
+      required: false
+    attributes:
+      label: Logs
+      description: Please paste any log errors.
+      placeholder: |
+        You can find the logs at...
+
+      render: bash
+
```

Comment:
```suggestion
```

Reply:
Can change the render to logs... I've added the rendering to encapsulate any text that might convert into undesirably links